File: problems/algorithms/array-operations/php/response/src/array-operations.php
<?php

function sum($array)
{
  $sum = 0;
  foreach ($array as $value) {
    $sum += $value;
  }
  return $sum;
}

function product($array)
{
  $product = 1;
  foreach ($array as $value) {
    $product *= $value;
  }
  return $product;
}

function sumOdds($array)
{
  $sum = 0;
  foreach ($array as $value) {
    if ($value & 1) {
      $sum += $value;
    }
  }
  return $sum;
}

File: problems/algorithms/array-operations/ruby/response/src/array_operations.rb
def sum(array)
  total = 0
  array.each do |numero|
    total += numero
  end
  return total
end

def product(array)
  total = 1
  array.each do |numero|
    total *= numero
  end
  return total
end

def sumOdds(array)
  total = 0
  array.each do |numero|
    if numero % 2 == 1
      total += numero
    end
  end
  return total
end
